templates/test/dev/cluster-template-custom-builds.yaml

```diff
@@ -375,6 +384,39 @@ spec:
           New-Item -ItemType Directory -Force -Path C:\tmp\
         path: C:/create-temp-folder.ps1
         permissions: "0744"
+      - content: |
+          Stop-Service kubelet -Force
+
+          $$binaries=@("kubeadm", "kubectl", "kubelet", "kube-proxy")
+          $$ci_url="https://${AZURE_STORAGE_ACCOUNT}.blob.core.windows.net/${JOB_NAME}/${KUBE_GIT_VERSION}/bin/windows/amd64"
+          foreach ( $$binary in $$binaries )
+          {
+            echo "installing package: $$binary ${KUBE_GIT_VERSION}"
+            curl.exe --retry 10 --retry-delay 5 "$$ci_url/$$binary.exe" --output "c:/k/$$binary.exe"
+          }
+
+          $$CONTAINERD_URL="${WINDOWS_CONTAINERD_URL}"
+          if($$CONTAINERD_URL -ne ""){
+            Stop-Service containerd -Force
+            echo "downloading containerd: $$CONTAINERD_URL"
+            curl.exe --retry 10 --retry-delay 5 -L "$$CONTAINERD_URL" --output "c:/k/containerd.tar.gz"
+            cmd /c tar -zxvf c:/k/containerd.tar.gz -C "c:/Program Files/containerd" --strip-components 1
+            containerd.exe --version
```

Comment:
Is there a reason to not output containerd hcsshim version always?
I think that info might be helpful (especially the shim version) which isn't available w/ `kubectl get node`

Reply:
The binary doesn't output a version as of now: https://github.com/microsoft/hcsshim/issues/1268

Reply:
So should line 405 be commented it out?
It is currently trying to run `containerd-shim-runhcs-v1.exe --version`

My initial comment in this thread was meant to suggest moving all of the --version calls outside of the conditional.

Reply:
updated